core/src/main/scala/com/databricks/labs/remorph/parsers/tsql/TSqlExpressionBuilder.scala

```diff
@@ -3,9 +3,10 @@ package com.databricks.labs.remorph.parsers.tsql
 import com.databricks.labs.remorph.parsers.tsql.TSqlParser._
 import com.databricks.labs.remorph.parsers.{FunctionBuilder, ParserCommon, intermediate => ir}
 import org.antlr.v4.runtime.Token
-import org.antlr.v4.runtime.tree.TerminalNode
+import org.antlr.v4.runtime.tree.{TerminalNode, Trees}
 
 import scala.collection.JavaConverters.asScalaBufferConverter
+import scala.collection.convert.ImplicitConversions.`collection AsScalaIterable`
```

Comment:
```suggestion
import scala.collection.JavaConverters._
```

Reply:
headOption cannot be used with that change. It seems the equivalent in JavaConverters is deprecated according to IntelliJ Ultimate at least.